.github/workflows/release.yml

```diff
@@ -1,8 +1,9 @@
 name: Release
 on:
-  push:
+#  push:
```

Comment:
I would prefer to remove the commented code.

Reply:
:heavy_check_mark: